guides/governance-guide/vip.md

```diff
@@ -18,33 +18,65 @@ Now that your wallet is connected, you can create a proposal. Look for the "Crea
 
-### Step 4: Choose the Proposal Type
+This action will open a modal presenting two options - "Upload file" and "Create Manually":
 
-After clicking the "Create Proposal" button, a new interface will pop up prompting you to select the proposal type. Select the relevant proposal type depending on the kind of change you want to propose.
+<figure><img src="../../.gitbook/assets/VIP-create-modal.png" alt=""><figcaption></figcaption></figure>
+
+Let's now take a look at each option and how they can used to create a VIP.
+
+### Step 4: Upload a proposal file
+
+The fastest way to create a proposal is by importing a VIP file. They are JSON formatted files containing all the information needed to create a VIP, such as the proposal's title, type, actions and parameters.
+
+Venus makes available a tool to generate VIP files, which you can find [here](https://github.com/VenusProtocol/vips). You can also check an [example of a valid proposal file here](https://github.com/VenusProtocol/venus-protocol-interface/blob/main/src/assets/proposals/vip-123.json). Note that all the fields will be validated according to their requirements and types.
+
+#### Step 4.1: Choose a VIP file
+
+Clicking "Upload file" will lead you to choose the proposal file you want to upload:
+
+<figure><img src="../../.gitbook/assets/VIP-upload.png" alt=""><figcaption></figcaption></figure>
+
+After uploading the desired file, it will be parsed and validated. Any possible errors found will be reported by the interface.
+
+#### Step 4.2: Confirm the Proposal
+
+Once the validation is done, the interface will present all the VIP data for your review:
+
+<figure><img src="../../.gitbook/assets/VIP-import-confirm.png" alt=""><figcaption></figcaption></figure>
+
+If everything looks good, click "Create" at the bottom of the modal to confirm the creation of your VIP.
+
+### Step 5: Create a Proposal Manually
+
+Instead of uploading a file, you can opt to manually create a proposal. By clicking on "Create manually", we are presented a sequence of screens:
+
+#### Step 5.1: Choose the Proposal Type
+
+After clicking the "Create manually" button, a new interface will pop up prompting you to select the proposal type. Select the relevant proposal type depending on the kind of change you want to propose.
 
 <figure><img src="../../.gitbook/assets/VIP_selectType.jpg" alt=""><figcaption></figcaption></figure>
 
-### Step 5: Enter Proposal Information
+#### Step 5.2: Enter Proposal Information
 
 The next step involves providing detailed information about your proposal. This includes the proposal's title, a brief description, and a link to an off-chain discussion related to your proposal. Ensure the information you provide is clear and concise to facilitate understanding by other community members.
 
 <figure><img src="../../.gitbook/assets/VIP3.jpg" alt=""><figcaption></figcaption></figure>
 
-### Step 6: Set the Voting Options
+#### Step 5.3: Set the Voting Options
```

Comment:
Rename the image for this section something like voting-options-5.3.jpg VIP4 is a bit confusing since this is section 5.3